controllers/transactions.js

```diff
@@ -0,0 +1,128 @@
+const { Transaction, Book } = require('../models')
+
+class transactionController {
```

Comment:
Inget nama kelas harus dimulai dengan huruf besar
```suggestion
class TransactionController {
```